app/audit/UserRegistrationSubmissionEvent.scala

```diff
@@ -47,23 +47,31 @@ object SubmissionEventDetail {
         }
       }
 
+      // TODO to ensure the auditing of the submission stays the same
+      // TODO change this to check if transId is None
```

Comment:
Are you happy leaving these TODOs in?
Seems pertinent to the story that this is still correct